core/consensus/babe/impl/block_executor.cpp

```diff
@@ -21,6 +21,8 @@ OUTCOME_CPP_DEFINE_CATEGORY(kagome::consensus, BlockExecutor::Error, e) {
   switch (e) {
     case E::INVALID_BLOCK:
       return "Invalid block";
+    case E::BLOCK_EXECUTOR_GONE:
+      return "Internal error";
```

Comment:
It's a bit suspicious how the constant name diverges with the message. Maybe a comment here would do good.

Reply:
Fixed